data/stateproofmsg/message.go

```diff
@@ -27,6 +27,10 @@ type Message struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
```

Comment:
Will be nice to add a few words to describe the purpose of this type `Message`.
Algod just creates it for external use.